docs/content/Coding Milestones/PoC2023q1/example1.md

```diff
@@ -52,7 +52,7 @@ nodes:
 For the guilder cluster, which will get two workloads, create a file
 named `guilder-config.yaml` with the following contents.  The workload
 that uses hostPort 8081 goes in both clusters, while the workload that
-uses hostPort 8082 goes only in the guilder cluster.
+uses hostPort 8097 goes only in the guilder cluster.
```

Comment:
This stays 8082.  Here `hostPort` is the kube jargon referring to the port on the node hosting the kube Pod, which is _inside_ the kind cluster not the host of the kind cluster.